metadata-ingestion/src/datahub/cli/docker_check.py

```diff
@@ -127,16 +208,26 @@ def check_local_docker_containers(preflight_only: bool = False) -> List[str]:
 
             if container.name in ENSURE_EXIT_SUCCESS:
                 if container.status != "exited":
-                    issues.append(f"{container.name} is still running")
+                    status = ContainerStatus.STILL_RUNNING
                 elif container.attrs["State"]["ExitCode"] != 0:
-                    issues.append(f"{container.name} did not exit cleanly")
+                    status = ContainerStatus.EXIT_FAILURE
 
             elif container.status != "running":
-                issues.append(f"{container.name} is not running")
+                status = ContainerStatus.DIED
             elif "Health" in container.attrs["State"]:
                 if container.attrs["State"]["Health"]["Status"] == "starting":
-                    issues.append(f"{container.name} is still starting")
+                    status = ContainerStatus.STARTING
                 elif container.attrs["State"]["Health"]["Status"] != "healthy":
```

Comment:
minor: are there any other states we should be aware of? since this will count any state aside from starting and healthy as "unhealthy"

Reply:
i haven't seen any other ones